src/main/java/org/ejbca/cvc/UnknownAccessRights.java

```diff
@@ -0,0 +1,26 @@
+package org.ejbca.cvc;
+
+import org.bouncycastle.util.encoders.Hex;
+
+public class UnknownAccessRights implements AccessRights {
```

Comment:
I think UnknownAccessRights sounds like an Exception, it took me some reading to understand it wasn't :-). Can you rename it to something that doesn't sound like an error like "GenericAccessRight" perhaps, or "RawAccessRight" perhaps it's good as it's a holder where you pass in "raw" values you might say?

Reply:
Good point! `RawAccessRight` seems like a conflict with `AuthorizationRoleRawValue` - but maybe I can just use that instead of the new type, there seems to be a lot of overlap